CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+### Added
+
+- Support for SQLAlchemy 2.0+
+- Added internal support for the new psycopg dialect.
```

Comment:
Can we do this change on another PR? So I can really focus on what is needed for SQLA v2

Reply:
Rolling back now